src/unix/os390.c

```diff
@@ -337,13 +350,17 @@ uint64_t uv_get_total_memory(void) {
 
 
 int uv_resident_set_memory(size_t* rss) {
-  W_PSPROC buf;
-
-  memset(&buf, 0, sizeof(buf));
-  if (w_getpsent(0, &buf, sizeof(W_PSPROC)) == -1)
-    return -EINVAL;
-
-  *rss = buf.ps_size;
+  void* psa;
+  void* ascb;
+  void* rax;
```

Comment:
void* arithmetic is the same as char* arithmetic according to z/os compiler.
But to be safe and conform with other platforms, I will change to char*